Services/data/templates/device-simulation-templates.json

```diff
@@ -0,0 +1,32 @@
+﻿[
```

Comment:
pleaxe rename the file to indicate this is an example, i.e. a template with multiple simulation